Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java

```diff
@@ -376,24 +301,6 @@ public void copyPerson() {
         saveMetadataAsBean(this.docStruct);
     }
 
-    /**
-     * Change current document structure.
-     *
-     */
-    public void changeCurrentDocstructType() {
-
-        if (this.docStruct != null && this.tempTyp != null) {
-            try {
-                DocStructInterface result = this.metaHelper.changeCurrentDocstructType(this.docStruct, this.tempTyp);
```

Comment:
Probably some methods in MetadataHelper class can be removed as method usages are removed here.

Reply:
Removed unused code from the MetadataHelper class in another commit.